product_docs/docs/edbcloud/beta/overview/03_security_compliance.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: "Security and Compliance"
+---
+
+### Security Governance
+
+- Cloud Security Compliance officer is responsible for maintaining compliance targets for their entire organization.
+- EDB Cloud infrastructure periodically undergoes SOC2 audit trials to ensure that EDB Cloud is always compliant with security requirements.
```

Comment:
We should hold off on talking about SOC2 until our audit is complete and report is issued.